src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -35,6 +54,13 @@ public Drivetrain(SwerveDrivetrainConstants driveTrainConstants, double Odometry
   public Drivetrain(SwerveDrivetrainConstants driveTrainConstants, SwerveModuleConstants... modules)
   {
     super(driveTrainConstants, modules);
+    // Create a photon camera and pose estimator object
+    m_photonCamera = new PhotonCamera("photonvision");
```

Comment:
I want to move away from default naming for both cameras choose names that make sense and implement now.